lib/Varien/Http/Adapter/Curl.php

```diff
@@ -192,8 +192,14 @@ public function read()
             $response = trim($response[1]);
         }
 
-        if (stripos($response, "Transfer-Encoding: chunked\r\n")) {
-            $response = str_ireplace("Transfer-Encoding: chunked\r\n", '', $response);
+        $responsePart = "HTTP/1.1 200 Connection established\r\n";
+        if (stripos($response, $responsePart)) {
```

Comment:
Because the response starts with "HTTP/1.1 200 Connection established\r\n",
`stripos` returns 0.

You need to test against `!== false`

Reply:
Or `=== 0`?

Reply:
It is very likely that it only appears at first position, so === 0 should be fine

we probably don't need to case-insensitive, so `str_starts_with` could be fine too

Reply:
Thought about it, but stripos is also used for ""Transfer-Encoding: chunked"